slam-libraries/viam-cartographer/src/main.cc

```diff
@@ -11,13 +11,13 @@
 #include "slam_service/slam_service.h"
 
 void exit_loop_handler(int s) {
-    LOG(INFO) << "Finishing session.\n";
+    LOG(INFO) << "Finishing session.";
```

Comment:
I might switch this and the system shutdown log to VLOG(0) so that users always see that (or change it to VLOG(1)

Reply:
yeah I tried that but it didn't show up in the log messages which is why I left it as is.